components/com_content/src/Controller/CategoryController.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+/**
+ * @package     Joomla.Site
+ * @subpackage  com_content
+ *
+ * @copyright   (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Content\Site\Controller;
+
+use Joomla\CMS\Access\Access;
+use Joomla\CMS\MVC\Controller\FormController;
+use Joomla\Utilities\ArrayHelper;
+
+// phpcs:disable PSR1.Files.SideEffects
+\defined('_JEXEC') or die;
+// phpcs:enable PSR1.Files.SideEffects
+
+/**
+ * Content category frontend controller.
+ *
+ * @since  6.1.0
+ */
+class CategoryController extends FormController
+{
+    /**
+     * Proxy for getModel to ensure the correct model is returned.
+     *
+     * @param   string  $name    The model name.
+     * @param   string  $prefix  The class prefix.
+     * @param   array   $config  Configuration array.
+     *
+     * @return  \Joomla\CMS\MVC\Model\BaseDatabaseModel|bool
+     *
+     * @since   6.1.0
+     */
+    public function getModel($name = 'CategoryForm', $prefix = 'Site', $config = [])
+    {
+        $config['ignore_request'] = false;
+
+        return parent::getModel($name, $prefix, $config);
+    }
+
+    /**
+     * The URL view item variable.
+     *
+     * @var    string
+     * @since  6.1.0
+     */
+    protected $view_item = 'categoryform';
+
+    /**
+     * The URL view list variable.
+     *
+     * @var    string
+     * @since  6.1.0
+     */
+    protected $view_list = 'categories';
+
+    /**
+     * The URL edit variable.
+     *
+     * @var    string
+     * @since  6.1.0
```

Comment:
```suggestion
     * @since   __DEPLOY_VERSION__
```

doing this means our build scripts will correctly set the version - it might not make it to 6.1 ;)